helm/templates/raw-spans-grouper-config.yaml

```diff
@@ -6,42 +6,27 @@ metadata:
     release: {{ .Release.Name }}
 data:
   application.conf: |-
-    flink.job {
-      {{- if hasKey .Values.rawSpansGrouperConfig.flink "job" }}
-      {{- if hasKey .Values.rawSpansGrouperConfig.flink.job "parallelism" }}
-      parallelism = {{ int .Values.rawSpansGrouperConfig.flink.job.parallelism }}
-      {{- end }}
-      {{- if hasKey .Values.rawSpansGrouperConfig.flink.job "metrics" }}
-      metrics {
-        metrics {
-          reporters = {{- toJson .Values.rawSpansGrouperConfig.flink.job.metrics.reporters | trim | nindent 12 }}
-        }
-      }
-      {{- end }}
-      {{- end }}
-    }
 
-    flink.source {
-      schema.registry {
-        schema.registry.url = "{{ .Values.rawSpansGrouperConfig.flink.source.schema.registry.url }}"
-        specific.avro.reader = true
-      }
-      kafka {
-        bootstrap.servers = "{{ .Values.rawSpansGrouperConfig.flink.source.kafka.bootstrapServers }}"
-      }
-    }
+    kafka.streams.config = {
```

Comment:
this looks like a lot of copy/pasta from https://github.com/hypertrace/raw-spans-grouper/pull/17/files#diff-204f96f3fcf2648351f042d9e4d698b2R13

would it be less different if variables were used instead?